deploy/olm-catalog/apicast-community-operator/apicast-community-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -168,4 +168,4 @@ spec:
   minKubeVersion: 1.11.0
   provider:
     name: Red Hat
-  version: 0.0.0
+  version: 0.1.0
```

Comment:
Set first version of the CSV to 0.1.0. Is that ok for you?

Reply:
`1.0.0`?